.github/workflows/linux_ci.yml

```diff
@@ -0,0 +1,40 @@
+on: [push]
+
+name: Linux CI
+
+jobs:
+  check:
+    name: Compile and Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install gtk3 and dependencies.
+        run: |
+          sudo apt update
+          sudo apt install \
+            libgtk-3-dev \
+            libxdo-dev
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+      
+      - name: Check Code
+        uses: actions-rs/cargo@v1
+        with:
+          command: check
+
+      - name: Run Clippy
+        uses: actions-rs/clippy-check@v1
```

Comment:
Instead of using this action, please just install clippy alongside the rust toolchain so we can remove this `token` thing. Take a look at how we do in `gtk3-rs` for example.

Reply:
I have used the following lines from gtk3-rs [as reference.](https://github.com/gtk-rs/gtk3-rs/blob/f559e5fa9778c60a673e358acc5bc65a02bc6115/.github/workflows/CI.yml#L70-L76)